WORKSPACE

```diff
@@ -37,11 +41,17 @@ http_archive(
 # Npm Install and Typescript Sass support #
 ###########################################
 load("@build_bazel_rules_nodejs//:index.bzl", "check_bazel_version", "npm_install")
+
 # The minimum bazel version to use with this repo is v3.1.0.
 check_bazel_version("3.1.0")
+
 # Install all dependencies with npm
 npm_install(
     name = "npm",
+    data = [
+        "//:patches/@angular+bazel+10.0.6.patch",
+        "//:patches/@bazel+typescript+2.0.3.patch",
+    ],
```

Comment:
Maybe also add `//:postinstall.js` and `//:view-engine-tsconfig.json` since it helped with build issues on my machine and could also affect others?

Reply:
I think we should do this in a separate PR but good point 👍